validphys2/src/validphys/pdfplots.py

```diff
@@ -1044,3 +1044,15 @@ def plot_lumi2d_uncertainty(pdf, lumi_channel, lumigrid2d, sqrts:numbers.Real):
     ax.grid(False)
 
     return fig
+
+
+class MixMatchPDFPlotter(BandPDFPlotter, ReplicaPDFPlotter):
+    """Special wrapper class to plot, in the same figure, PDF bands and PDF replicas
+    depending on the type of PDF.
+    Practical use: plot together the PDF central values with the NNPDF bands
+    """
+    def draw(self, pdf, grid, flstate):
+        if pdf.label.startswith("HOP"):
```

Comment:
```suggestion
    def draw(self, pdf, grid, flstate):
        # small values corresponding to charm result in a +1 placed on top of
        # the y-axis if axes.formatter.useoffset is not False
        plt.rcParams['axes.formatter.useoffset'] = False 
        if pdf.label.startswith("HOP"):
```

Reply:
Should probably make that a default.

Reply:
You mean in the .mpl files? Maybe a good idea, but I'm not sure this PR will ever be merged so maybe that should be done separately.

Reply:
Yes indeed.